packages/flutter_image/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## NEXT
+## 4.2.0
```

Comment:
This is not a new feature; it should be 4.1.11

Reply:
Yes I tried that before but the test failed saying `Incorrectly updated version`. Like it is failing now.

Reply:
Yes, that's because before the latest version was marked as `NEXT` so I changed it to `4.1.10` and marked mine as `4.1.11`. So should I add both mine and `4.1.10` as one?

Reply:
Yes, please see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#updating-a-changelog-that-has-a-next

Reply:
Done.